webapp/channels/src/components/trial_benefits_modal/trial_benefits_modal.test.tsx

```diff
@@ -130,7 +130,7 @@ describe('components/trial_benefits_modal/trial_benefits_modal', () => {
             </Provider>,
         );
 
-        wrapper.find(GenericModal).props().onExited();
+        wrapper.find(GenericModal).props().onExited?.();
```

Comment:
Why are all these being made nullable now? The original tests don't seem very useful because they just call something and check if it was called, but I would expect that to fail if `.props().onExited` is undefined

Reply:
Ah, I see. I forgot that there were cases where modals aren't managed by the `ModalController` since that always passes an `onExited` so that it can unmount the component after it animates out